src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -155,6 +156,20 @@ module BABYLON {
             return this._teleportationTarget;
         }
 
+        /**
+        * Set if VR Controllers / Gamepad inputs are enabled to teleport camera
+        */
+        public set teleportationInputsEnabled(value: boolean) {

```

Comment:
I'm wondering if we could not simply use this_teleportationEnabled

Reply:
@deltakosh  this is something I discussed with @RaananW. _teleportationEnabled is used as a flag to control if teleportation code was initialized inside the public method enableTeleportation. You can have collision if you set first teleportationEnabled = true and then invoke enableTeleporation as it won't initialize anything. The other flag, _teleportationAllowed, gets updated every time the target mesh collided by the ray is not the floor mesh/meshes configured so is not a flag to use in this case.

I studied reusing the existing flags but they are not for the same purpose, thats what I decided to create a flag related to the inputs, that are the ones being disabled / enabled.

I'll do whatever you consider better.

Reply:
I would say it is just a matter of naming here
Perhaps we should rename _tepelortationEnabled to _teleportationInitialized and then reuse teleportationEnabled